spec/rubocop/cop/rails/transaction_exit_statement_spec.rb

```diff
@@ -43,4 +43,14 @@
       end
     RUBY
   end
+
+  it 'does not register an offense when `raise` is used in `loop` in transactions' do
```

Comment:
Shouldn’t this refer to `break` rather than `raise`?

```suggestion
  it 'does not register an offense when `break` is used in `loop` in transactions' do
```

Reply:
Oops! Thank you!